src/Sulu/Bundle/AdminBundle/Resources/js/components/Input/Input.js

```diff
@@ -128,6 +128,7 @@ export default class Input<T: ?string | ?number> extends React.PureComponent<Inp
                     {!loading && icon &&
                         <div className={prependContainerClass}>
                             <Icon
+                                // maybe add an aria-label here?
```

Comment:
There is already a `aria-label` on it rendered by the Icon component. Which should be more removed as it is totally false here. But at current state you can use `aria-label="su-envelope"` selector if there is no other way.

Reply:
Just for your information `onClick` on the Icon was a false decision and ideal case the click would be handled not in the Icon component and so it would be something like this:

```
const iconElement = <Icon name={name} className={className} style={iconStyle} />

onIconClick
   ? <button aria-label={iconClickLabel} type="button" onClick={onIconClick}>
       {iconElement}
   </button> 
   : iconElement
```

but at current state we can not change it. And you can go on the existing: `aria-label="su-envelope"`

Reply:
Ah, I forgot the Icon component itself already had an `aria-label` on the span.